pep-0678.rst

```diff
@@ -114,36 +119,39 @@ includes a note of the minimal failing example::
 
 Non-goals
 ---------
-``__note__`` is *not* intended to carry structured data.  If your note is for
-use by a program rather than display to a human, `we recommend
+Tracking multiple notes as a tuple, rather than by concatenating strings when
+notes are added, is intended to maintain the distinction between strings
+which might be translated by packages such as ``friendly-traceback``.
```

Comment:
```suggestion
notes are added, is intended to maintain the distinction between the
individual notes, which might be required in specialized use cases, such 
as translation of the notes by packages like as ``friendly-traceback``.
```

Reply:
```suggestion
notes are added, is intended to maintain the distinction between the
individual notes. This might be required in specialized use cases, such 
as translation of the notes by packages like ``friendly-traceback``.
```

Based on @iritkatriel 's suggestion, but splitting it into two sentences to avoid a run-on, excessive commas and a hard to parse structure. I've also fixed an editing typo.